api/constants.js

```diff
@@ -0,0 +1,5 @@
+const UNSUSCRIBETEXT = {
+    body: 'We have received your request to unsubscribe from SetLife emails and this has been automaticallyprocessed in our system. After this email we wont bother you again! If for some reason you continue to receive emails, we apologize in advance and ask that you please send a message to contact@setlife.network so we can resolve this manually. Thanks and have a great day!'
+}
+
+export default { UNSUSCRIBETEXT }
```

Comment:
Alternatively you can export variables like this:
```
export const UNSUSCRIBETEXT = {
    body: 'We have received your request to unsubscribe from SetLife emails and this has been automaticallyprocessed in our system. After this email we wont bother you again! If for some reason you continue to receive emails, we apologize in advance and ask that you please send a message to contact@setlife.network so we can resolve this manually. Thanks and have a great day!'
}
```

but again just another FYI